.changeset/light-candles-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+'@celo/celocli': patch
+---
+
+Test changes: separate L1 and L2 tests, change stripAnsiCodes to stripAnsiCodesAndTxHashes that now strips transaction hashes as well
```

Comment:
this is an internal dev change not user facing, which means no changeset needed.